Makefile

```
@@ -40,16 +40,18 @@ test:
 test-all:
 	tox
 
+validate-docs:
+	./newsfragments/validate_files.py
+	towncrier build --draft --version preview
+
 build-docs:
 	rm docs/eth_hash.backends.rst
 	sphinx-apidoc -o docs/ . setup.py "*conftest*"
 	$(MAKE) -C docs clean
 	$(MAKE) -C docs html
 	$(MAKE) -C docs doctest
-	./newsfragments/validate_files.py
-	towncrier --draft --version preview
 
-docs: build-docs
+docs: validate-docs build-docs
```

Validate should be after building here:

```suggestion
docs: build-docs validate-docs
```